.travis.yml

```diff
@@ -3,7 +3,6 @@ notifications:
 
 language: python
 
-cache: pip
```

Comment:
why'd you remove this?

Reply:
olvide agregarlo de nuevo, ahora lo agrego